src/index.js

```diff
@@ -1,26 +1,29 @@
 import Vue from "vue";
 import VueRouter from "vue-router";
+import "animate.css";
```

Comment:
Maybe sort these imports here so Vue-related imports go in one chunk, css imports in another, and js imports in yet another chunk.

Reply:
Sure thing, changed.